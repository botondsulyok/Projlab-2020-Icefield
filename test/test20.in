field spawn 0
entity exp Explorer spawn
ability spawn exp
stats spawn