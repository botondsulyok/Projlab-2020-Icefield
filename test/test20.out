The iceunit is stable!
id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: exp
pullers: exp
masters: exp
neighbours: null

